src/main/java/nl/tudelft/contextproject/logging/LogFormatter.java

```diff
@@ -31,23 +31,13 @@
 	public LogFormatter(String format) {
 		this.format = format;
 	}
-	
-    /**
-     * Formats the given LogRecord with the format set in this LogFormat.
-     * 
-     * @param record
-     * 		the LogRecord to format
-     * 
-     * @return
-     * 		a String representation of the record, using the format of this LogFormatter.
-     */
```

Comment:
I accidentally deleted the wrong comment because I misinterpreted your comment. My apologies.